docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -67,6 +67,20 @@ abstract class Client {
     topic: string,
     encryptedMessage: string,
   }): Promise<NotifyNotification>;
+  
+
+  // Generates and returns incomplete CacaoPayload
+  // To be handled in recaps based authentication.
+  // This enables one-click-auth based authentication to
+  // also enable permissions for notify API.
+  public abstract prepareRegistrationWithRecaps(params: {
+    domain: string;
+    allApps?: boolean;
+  }): Promise<{
+    allApps: boolean;
```

Comment:
do we need this boolean in the response?

Reply:
It's needed for javascript because of how this will be used by dapps, I imagine the same is true for Swift and Kotlin dapps.